src/main/java/com/amplitude/HttpTransport.java

```diff
@@ -89,6 +89,10 @@ public void setCallbacks(AmplitudeCallbacks callbacks) {
           Response response = null;
           try {
             response = httpCall.makeRequest(events);
+            if (logger.getLogMode() == AmplitudeLog.LogMode.DEBUG) {
```

Comment:
We already have a logMode check right? Then we don't need to do this check here. I was thinking you can just call logger.debug(...)